docs/source/onnx_dynamo.rst

```diff
@@ -149,8 +150,41 @@ The main advantages are:
 
    generated/onnx_dynamo_diagnostics_rules/*
 
+Known issues
+============
+
+torch.jit.instance
+------------------
+
+The exporter fails because `torch.jit.isinstance` was used instead of `isinstance`
+and it is not supported by :func:`torch.export.export`.
+If modifying the code is not possible, replacing
+`torch.jit.isinstance` by instance `isinstance` makes the export work:
+
+.. code-block:: python
+
+    @contextlib.contextmanager
+    def bypass_export_jit_isinstance():
+        """
+        Tries to bypass some functions torch.export.export does not
+        support such as ``torch.jit.isinstance``.
+        """
+        import torch.jit
+
+        f = torch.jit.isinstance
+        torch.jit.isinstance = isinstance
+
+        try:
+            yield
+        finally:
+            torch.jit.isinstance = f
+
+
+        with bypass_export_jit_isinstance():
```

Comment:
Should we just do it for users?

Reply:
I let you decide.

Reply:
I think we should do this for the users as there's no downside to it (as far as I can tell?) I can take this contextmanager to the capture_strategies file